### ISSUE TITLE
Add to documentation to never ever use Model.prototype.save

### ISSUE THREAD

**Issue by kennethlynne** [clarkbw/loopback-ds-timestamp-mixin#17]
I guess the validateUpsert=false setting is the culprit for this issue?

https://github.com/strongloop/loopback/issues/1714#issuecomment-147889351

**Comment by clarkbw** [comment]
Yes, there isn't an easy way around the validate upsert issue. Now that I'm back I'll update the docs to better explain the situations that can arise.
